ios pod update, nfc modal implementation, merge with stepper updates

merveillevaneck opened the pull request.

merveillevaneck committed:
ios pod update, nfc modal implementation, merge with stepper updates

BtencateSphereon commented on src/assets/images/ausweis_icon.png:
i would suggest adding images as svg

BtencateSphereon commented on src/localization/translations/en.json:
```diff
@@ -93,6 +93,9 @@
   "terms_of_service_decline_action_caption": "Exit app",
   "terms_of_service_title": "Terms of service",
   "terms_of_service_subtitle": "This document was last updated on January 5, 2023.",
+  "import_data_title": "Import personal data",
```
where are the dutch localizations? it is fine adding them to the dutch file in english for now. so no need to translate it to dutch. but just make sure thy are present so we know which ones we still need to translate

BtencateSphereon commented on src/screens/Onboarding/ImportPersonalDataSCreen/index.tsx:
```diff
@@ -0,0 +1,322 @@
+  focusOnMount?: boolean;
+};
+
+export const AusweisEPinModal = (props: AusweisEPinModalProps) => {
```
please move this to it's own file

BtencateSphereon commented on src/screens/Onboarding/ImportPersonalDataSCreen/index.tsx:
```diff
@@ -0,0 +1,322 @@
+  onScan: () => void;
+};
+
+export const AusweisScanModal = (props: AusweisScanModalProps) => {
```
please move this to it's own file

BtencateSphereon reviewed (changes requested):
the ticket is about implementing whole step 3, so we are missing some screens and implementation in this PR

merveillevaneck committed:
(WIP) added UI fo import personal data consent step

odididi committed:
Small fixes; you can maybe add them to the correct commits

merveillevaneck committed:
added more screens to NFC UI

- added placheolder screen for local auth step on nfc flow
- added data loader screen on nfc flow
- added final data verification screen on nfc flow
- created a component for showing requested information summary
- updated navigation and machine state to handle navigation to new screens

merveillevaneck commented:
closing because of syncing updates with newly merged `feat/new-onboarding` so rebased on top of funke and continued there addressing above comments as well as adding more UI

merveillevaneck committed:
ran prebuild on ios for ausweis sdk integration

odididi committed:
Fixes in import data consent screen

odididi committed:
wip-scan-card

Commit:
fixed footer below scroll container
